ios/MagnetScannerReact.swift

```diff
@@ -24,16 +24,23 @@ import Foundation
   }
   
   func onItemFound(item: Dictionary<String, AnyObject>) {
-    print("item found", item["url"])
+    debugPrint("item found", item["url"])
```

Comment:
`print` is executed no matter what, even in a production build. `debugPrint` otherwise will work in our debug builds but won't do anything on a production one.